package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Actions Http Client",
   "main": "index.js",
   "scripts": {
-    "build": "rm -Rf ./_out && tsc && cp package*.json ./_out && cp *.md ./_out && cp LICENSE ./_out && cp actions.png ./_out",
+    "build": "tsc",
```

Comment:
This wasn't working on my Windows machine (`rm` not found) and it looks like this is combining `build` and some release since `LICENSE` is copied over.